### ISSUE TITLE
Unable to install under Fedora 30

### ISSUE THREAD

**Issue by zdm** [rurban/Cpanel-JSON-XS#153]
````
t/116_incr_parse_fixed.t ... ok
t/117_numbers.t ............ ok
t/118_type.t ............... 229/381
#   Failed test at t/118_type.t line 240.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 241.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 242.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 243.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 244.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 245.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 252.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 253.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 254.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 255.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 256.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 257.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 288.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 289.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'

#   Failed test at t/118_type.t line 303.
#          got: '1.18973149535723177e+4932'
#     expected: '1.79769313486232e+308'

#   Failed test at t/118_type.t line 304.
#          got: '-1.18973149535723177e+4932'
#     expected: '-1.79769313486232e+308'
# Looks like you failed 16 tests of 381.
t/118_type.t ............... Dubious, test returned 16 (wstat 4096, 0x1000)
Failed 16/381 subtests
t/119_type_decode.t ........ ok
t/11_pc_expo.t ............. ok
t/12_blessed.t ............. ok
t/13_limit.t ............... ok
t/14_latin1.t .............. ok
t/15_prefix.t .............. ok
t/16_tied.t ................ ok
```

**Comment by rurban** [comment]
32bit? Which arch?

**Comment by zdm** [comment]
64 bit, the is already no 32 bit distribution.

```
Summary of my perl5 (revision 5 version 30 subversion 0) configuration:

  Platform:
    osname=linux
    osvers=5.2.7-200.fc30.x86_64
    archname=x86_64-linux-ld
    uname='linux e3f3884c0a4d4b91ac6eb54781e3aba4 5.2.7-200.fc30.x86_64 #1 smp thu aug 8 05:35:29 utc 2019 x86_64 x86_64 x86_64 gnulinux '
    config_args='-des -Dusemorebits -Duselargefiles -Dprefix=/usr/perlbrew/perls/perl-5.30.0 -Duserelocatableinc -Dman1dir=none -Dman3dir=none'
    hint=recommended
    useposix=true
    d_sigaction=define
    useithreads=undef
    usemultiplicity=undef
    use64bitint=define
    use64bitall=define
    uselongdouble=define
    usemymalloc=n
    default_inc_excludes_dot=define
    bincompat5005=undef
  Compiler:
    cc='cc'
    ccflags ='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2'
    optimize='-O2'
    cppflags='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include'
    ccversion=''
    gccversion='9.2.1 20190827 (Red Hat 9.2.1-1)'
    gccosandvers=''
    intsize=4
    longsize=8
    ptrsize=8
    doublesize=8
    byteorder=12345678
    doublekind=3
    d_longlong=define
    longlongsize=8
    d_longdbl=define
    longdblsize=16
    longdblkind=3
    ivtype='long'
    ivsize=8
    nvtype='long double'
    nvsize=16
    Off_t='off_t'
    lseeksize=8
    alignbytes=16
    prototype=define
  Linker and Libraries:
    ld='cc'
    ldflags =' -fstack-protector-strong -L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib /lib/../lib64 /usr/lib/../lib64 /lib /lib64 /usr/lib64 /usr/local/lib64
    libs=-lpthread -lgdbm -ldb -ldl -lm -lcrypt -lutil -lc -lgdbm_compat
    perllibs=-lpthread -ldl -lm -lcrypt -lutil -lc
    libc=libc-2.29.so
    so=so
    useshrplib=false
    libperl=libperl.a
    gnulibc_version='2.29'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs
    dlext=so
    d_dlsymun=undef
    ccdlflags='-Wl,-E'
    cccdlflags='-fPIC'
    lddlflags='-shared -O2 -L/usr/local/lib -fstack-protector-strong'


Characteristics of this binary (from libperl):
  Compile-time options:
    HAS_TIMES
    PERLIO_LAYERS
    PERL_COPY_ON_WRITE
    PERL_DONT_CREATE_GVSV
    PERL_MALLOC_WRAP
    PERL_OP_PARENT
    PERL_PRESERVE_IVUV
    USE_64_BIT_ALL
    USE_64_BIT_INT
    USE_LARGE_FILES
    USE_LOCALE
    USE_LOCALE_COLLATE
    USE_LOCALE_CTYPE
    USE_LOCALE_NUMERIC
    USE_LOCALE_TIME
    USE_LONG_DOUBLE
    USE_PERLIO
    USE_PERL_ATOF
  Built under linux
  Compiled at Oct 13 2019 14:18:56
```

**Comment by rurban** [comment]
nvtype='long double', I thought so. 
That needs to be adjusted in the tests only, the module itself is fine. you can force install it.

**Comment by zdm** [comment]
Got it, thanks.

**Comment by rurban** [comment]
Fixed with 4.14
